config/crd/v1/bases/velero.io_backups.yaml

```diff
@@ -479,12 +479,15 @@ spec:
                 type: string
               uploaderConfig:
                 description: UploaderConfig specifies the configuration for the uploader.
-                nullable: true
                 properties:
                   parallelFilesUpload:
                     description: ParallelFilesUpload is the number of files parallel
                       uploads to perform when using the uploader.
                     type: integer
+                  writeSparseFiles:
```

Comment:
Is this available for backups?

Reply:
I've updated the API, which would not include the `writeSparseFiles` in backup